.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# Invenio-Audit-Logs is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+
+name: Publish
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build-n-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel babel
+      - name: Build package
+        run: |
+          python setup.py compile_catalog sdist bdist_wheel
+      - name: pypi-publish
+        uses: pypa/gh-action-pypi-publish@v1.3.1
+        with:
+          user: __token__
+          
+          password: ${{ secrets.pypi_token }}
+          
```

Comment:
```suggestion
name: Publish

on:
  push:
    tags:
      - v*

jobs:
  build-n-publish:
    uses: inveniosoftware/workflows/.github/workflows/pypi-publish.yml@master
    secrets: inherit
```

Reply:
good point @Samk13 , I think module cookiecutter needs a little update

Reply:
Sure, we can look into this!